index.d.ts

```diff
@@ -85,23 +133,21 @@ declare module "react-native-twilio-video-webrtc"{
   type iOSConnectParams = {
     accessToken: string;
     roomName?: string;
-    encodingParameters?: {
-      enableH264Codec?: boolean;
-      // if audioBitrate OR videoBitrate is provided, you must provide both
-      audioBitrate?: number;
-      videoBitrate?: number;
-    }
-  }
+    enableH264Codec?: boolean;
+    // if audioBitrate OR videoBitrate is provided, you must provide both
+    audioBitrate?: number;
+    videoBitrate?: number;
+  };
```

Comment:
I'm wondering why did you extract fields outside of `encodingParameters`?

Reply:
I was wondering the same...

Reply:
I don't see a reason for this nesting. All connection options can be combined in one map.